packages/blueprint-mongodb/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@onehilltech/blueprint-mongodb",
   "description": "Blueprint.js module for MongoDB",
-  "version": "4.75.1",
+  "version": "4.75.2",
```

Comment:
@centatmoney Do not change the version number.

Reply:
Sorry, I undid that change.